core/ui-common/src/main/kotlin/com/wire/android/ui/common/bottomsheet/WireModalSheetLayout.kt

```diff
@@ -93,3 +94,44 @@ fun MenuModalSheetContent(
         buildMenuSheetItems(items = menuItems)
     }
 }
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun WireModalSheetLayout2(
```

Comment:
suggestion: maybe mark previous one as deprecated so that it's clear that this one is favourable 😄